mobile-widgets/qml/DownloadFromDiveComputer.qml

```diff
@@ -467,7 +467,7 @@ Kirigami.Page {
 					importModel.recordDives()
 					// it's important to save the changes because the app could get killed once
 					// it's in the background - and the freshly downloaded dives would get lost
-					manager.saveChangesLocal()
```

Comment:
Note that this does *not* the same (except on iOs). `changesNeedSaving()` saves to the cloud, whereas `saveChangesLocal()` saves only locally. Not sure if you intend that.

Reply:
it saves to the cloud if auto sync is enabled.
So yes, that's what I intended.
I think :-)

Reply:
Ok - Then, while your at it, please make saveChangesLocal() private to QMLManager, as I think that was the last external caller. When doing the call graph I thought there was a reason for doing this differently here.

Reply:
A allowed myself to make that change and force-push. Please pull before going ahead to avoid complications.